src/main/resources/db/changelog/20220722000000-add_delete_cascade_to_consumer_fks.xml

```diff
@@ -6,56 +6,93 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
-    <changeSet id="20220613000000-1" author="jalbrech">
+    <changeSet id="20220722000000-1" author="jalbrech">
+        <preConditions onFail="MARK_RAN">
```

Comment:
With the removal and recreation of the foreign keys in the same changeset, we'll correctly remove the FK only if it exists and rebuild it as we expect, but if a system somehow doesn't have the FK at all, we won't attempt to build one for them.

Reply:
For each FK I added a change set for removing the existing FK if it exists and another change set for adding the new FK with the cascade on delete.